EjercicioClases/Model/TransactionRegistry.pm

```diff
@@ -4,7 +4,7 @@ use strict;
 use warnings;
 use Data::Dumper;
 
-
+use Controller::DatabaseController;
 
 sub new{
     my $class = shift;
```

Comment:
Please consider using hash structures to send/receive multiple sub parameters in the `my ($s,%args) = @_` way. For example:
`my ($class, %args) = @_;`